config/core/200-roles/addressable-resolvers-clusterrole.yaml

```diff
@@ -27,7 +27,7 @@ aggregationRule:
   clusterRoleSelectors:
   - matchLabels:
       duck.knative.dev/addressable: "true"
-rules: [] # Rules are automatically filled in by the controller manager.
+# rules: [] # Rules are automatically filled in by the controller manager.
```

Comment:
Should we just remove the line rather than commenting it?

Reply:
Done.

Didn't have much of an opinion on it - just seemed like the initial author wanted to communicate something so I opted to keep the information. I also can agree with just removing.